fetch.bs

```diff
@@ -3375,8 +3374,8 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
 
   <dl class=switch>
    <dt><var>request</var>'s <a for=request>current URL</a>'s <a for=url>origin</a> is
-   <a>same origin</a> with <var>request</var>'s <a for=request>origin</a>, <var>request</var>'s
-   <a for=request>tainted origin flag</a> is unset, and the <i>CORS flag</i> is unset
+   <a>same origin</a> with <var>request</var>'s <a for=request>origin</a>, and <var>request</var>'s
+   <a for=request>response tainting</a> is "<code>basic</code>"
```

Comment:
It's likely I'm missing something, but I don't exactly see how this condition changes, could you please explain:

> This also fixes an issue whereby we checked origin tainting rather than response tainting in main fetch which results in a minor bug when it comes to opaquing A -> B -> A chains.

From what I can tell, assuming the current URL's origin is same-origin with request's origin, then:
 - Whenever _tainted origin flag_ is unset, _response tainting_ is always "basic" here, and
 - Whenever _tainted origin flag_ is set, _response tainting_ is not "basic"

Reply:
I think you're right. A -> B -> A does set the tainted origin flag. I guess that means this whole change is editorial. I still think it's for the better though.

Reply:
I agree, I think favoring response tainting is clearer. Thanks.